Flow.Launcher/HotkeyControl.xaml.cs

```diff
@@ -103,7 +103,7 @@ public async Task SetHotkey(HotkeyModel keyModel, bool triggerValidate = true)
 
         public void SetHotkey(string keyStr, bool triggerValidate = true)
         {
-            SetHotkey(new HotkeyModel(keyStr), triggerValidate);
+            _ = SetHotkey(new HotkeyModel(keyStr), triggerValidate);
```

Comment:
I don't think it makes sense to have different return value for these two methods. Could you make it also either async void or async Task.